### PR TITLE
Upgrade to v8

### PR DESCRIPTION
**Pull request by kunalkapadia**

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -4,6 +4,7 @@
     "es6": true,
```
Comment (BadgerBadgerBadgerBadger) [inline .eslintrc]: @KunalKapadia Weren't we using this? https://github.com/Codigami/cf-hapi-starter-kit/blob/master/.eslintrc

File: Dockerfile [hunk 2]
```diff
@@ -17,7 +17,7 @@ RUN yarn --production --pure-lockfile
 COPY . /app/
 
 #expose default port of the docker to 3000
-EXPOSE 3000
+EXPOSE 3030
```
Comment (BadgerBadgerBadgerBadger) [inline Dockerfile:20]: @KunalKapadia Why this? Just curious.

File: index.js [hunk 3]
```diff
@@ -36,23 +35,20 @@ process.on('SIGINT', gracefulStopServer)
...
   }
-).catch((err) => {
-  logger.error(err)
-  process.exit(1)
-})
+}
+
+startServer()
+
+
```
Comment (BadgerBadgerBadgerBadger) [inline index.js]: @KunalKapadia Two newlines.

**Comment by BadgerBadgerBadgerBadger** [comment]
Suggestions:

- What do you think about adding the [debug](https://github.com/visionmedia/debug) package and making use of that?
- In logger.js, we can reference the app name from the config instead of the package.
- `.*` in `.dockerignore` will cause the `.eslintrc` and `.istanbul.yml` to not be copied into the docker instance. Linting and testing wouldn't use these configs.

File: Dockerfile (hunk 2 above)
Comment (kunalkapadia) [reply Dockerfile:20]: @ScionOfBytes `3000` is quite common port and generally not available. For example aerospike by default runs on 3000. So decided to use any uncommon port.

File: .eslintrc (hunk 1 above)
Comment (kunalkapadia) [reply .eslintrc]: @ScionOfBytes I don't how older file got cloned. Let me update to `standard`.

File: index.js (hunk 3 above)
Comment (kunalkapadia) [reply index.js]: @ScionOfBytes Will fix this.

**Comment by kunalkapadia** [comment]
@ScionOfBytes 
• What do you think about adding the debug package and making use of that?
   A. `debug` is generally recommended for libraries only, for application either winston or bunyan is used as they offer more features. In bunyan also you can set logLevel and do selective debugging.
Let me know if you have more thoughts on this.
Reference: https://blog.risingstack.com/node-js-logging-tutorial/

• In logger.js, we can reference the app name from the config instead of the package.
   A. Makes sense. Updated.

• .* in .dockerignore will cause the .eslintrc and .istanbul.yml to not be copied into the docker instance. Linting and testing wouldn't use these configs.
   A. Have updated .dockerignore to copy .eslintrc and .istanbul.yml

**Pull request merged**